elements/pfe-card/src/pfe-card.js

```diff
@@ -82,20 +82,30 @@ class PfeCard extends PFElement {
     return "pfe-card";
   }
 
-  get styleUrl() {
-    return "pfe-card.scss";
+  get schemaUrl() {
+    return "pfe-card.json";
   }
 
   get templateUrl() {
     return "pfe-card.html";
   }
 
+  get styleUrl() {
+    return "pfe-card.scss";
+  }
+
+  get imageSrc() {
+    return this.getAttribute("pfe-img-src");
+  }
+
   get backgroundColor() {
-    return this.getAttribute("color") || "base";
+    return (
+      this.getAttribute("pfe-color") || this.getAttribute("color") || "base"
+    );
   }
 
   static get observedAttributes() {
-    return ["color"];
+    return ["pfe-color", "pfe-img-src", "pfe-size"];
```

Comment:
@castastrophe I'm curious about the pfe-size=small attribute. Why not set this as a boolean? Would we want to leave it open for other size values later?

Reply:
Yes and I wanted to match the approach we've taken in other components too